packages/components/src/Account/Auth/AuthStyledComponents.tsx

```diff
@@ -50,13 +50,13 @@ export const ConfirmedEmailLetterIcon = styled(CheckEmailLetterIcon)`
   background-image: url(${confirmedEmailImage});
 `;
 
-export const CenterWrapper: React.SFC = ({ children }) => (
+export const CenterWrapper: React.StatelessComponent = ({ children }) => (
```

Comment:
Should these be `React.FunctionComponent`?

Reply:
weird good catch. I did a global search and replace, not sure how that one got missed. This should still work, but will say its deprecated in when building. will fix